.eslintrc

```diff
@@ -23,6 +23,8 @@
     "sourceType": "module"
   },
   "globals": {
+    "INTERSECTION_OBSERVER_POLYFILL": "readonly",
+    "INTERSECTION_OBSERVER_POLYFILL_INABOX": "readonly",
```

Comment:
Note to self: Make this dynamic as described in #25227. (Not blocking for this PR.)